tests_phi/testcases/manage_pages.py

```diff
@@ -0,0 +1,100 @@
+    def test_da_mp_tc021(self):
+        ''' Verify that user is able to edit the name of the page (Parent/Sibbling) successfully'''
+         
+        logging.info("set variables")
+        page_name_1 = self.common_methods.random_string(10)
+        page_name_2 = self.common_methods.random_string(10)
+        page_name_3 = self.common_methods.random_string(10)
+        page_name_4 = self.common_methods.random_string(10)
+         
+        logging.info("1. Login to Dashboard and login")
+        self.login_page.go_to_login_page()
+        self.login_page.login(constants.USERNAME,"")
+         
+        logging.info("2. Add page 1")
+        self.dashboard_page.add_page(page_name_1)
+         
+        logging.info("3. Add page 2")
+        self.dashboard_page.add_page(page_name_2, page_name_1)
+         
+        logging.info("4. Open Page 1")
+        self.dashboard_page.open_page("Overview/"+page_name_1)
+         
+        logging.info("5. Edit Page 1 to other name")
+        self.dashboard_page.edit_page(page_name_3)
+         
+        logging.info("VP: User is able to edit the name of parent page successfully")
+        assert page_name_3 in browser.get_driver().title
+         
+        logging.info("6. Open Page 2")
+        self.dashboard_page.open_page("Overview/"+page_name_3+"/"+page_name_2)
+         
+        logging.info("7. Edit Page 2 to other name")
+        self.dashboard_page.edit_page(page_name_4, page_name_3)
+         
+        logging.info("VP: User is able to edit the name of parent page successfully")
+        assert page_name_4 in browser.get_driver().title
+        
+        self.path_page_name_2 = "Overview/"+page_name_3+"/"+page_name_4
+        self.path_page_name_1 = "Overview/"+page_name_3
+        
+    @pytest.mark.usefixtures("clean_up_tc022")
+    def test_da_mp_tc022(self):
+        '''Verify that user is unable to duplicate the name of sibbling page under the same parent page'''
+         
+        logging.info("set variables")
+        page_name_1 = self.common_methods.random_string(10)
+        incorrect_login_message = " already exists. Please enter a different name."
+         
+        logging.info("1. Login to Dashboard and login")
+        self.login_page.go_to_login_page()
+        self.login_page.login(constants.USERNAME,"")
+         
+        logging.info("2. Add page 1")
+        self.dashboard_page.add_page(page_name_1)
+         
+        logging.info("3. Add other page with the same name with page 1")
+        self.dashboard_page.select_global_selecting_item("Add Page")
+        self.dashboard_page.enter_page_info(page_name_1)
+        self.dashboard_page.click_button_on_add_edit_modal("OK")
+         
+        logging.info("VP: '%s' message displays"% incorrect_login_message)
+        assert browser.get_alert().text == page_name_1 + incorrect_login_message
+        browser.close_alert()
+        self.dashboard_page.click_button_on_add_edit_modal("Cancel")
```

Comment:
`click_button_on_add_edit_modal` should be `dismiss_edit_modal`

Reply:
updated

Reply:
updated